.github/workflows/vulnz-docker-release.yml

```diff
@@ -0,0 +1,46 @@
+name: docker pr
+
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  IMAGE_FQDN: ghcr.io/jeremylong/open-vulnerability-data-mirror
+  VERSION: ${{ github.ref_name }}
+
+jobs:
+  docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ github.token }}
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+      - name: Run build
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
GH token should not be needed for the build
```suggestion
```

Reply:
removed